data/src/main/java/com/sincalexandrudaniel/data/feature/news/local/NewsEntity.java

```diff
@@ -0,0 +1,43 @@
+package com.sincalexandrudaniel.data.feature.news.local;
+
+import androidx.annotation.Nullable;
+import androidx.room.Entity;
+import androidx.room.PrimaryKey;
+
+@Entity(tableName = "newsTable")
+public class NewsEntity {
```

Comment:
let's: 
- remove all the setter and getters, as we don't want to encapsulate anything, the opposite (no logic and no inheritance)
- let's make all the filed final so that we get an immutable class (adding a constructor)